mlrun/datastore/base.py

```diff
@@ -104,27 +121,28 @@ def download(self, key, target_path):
     def upload(self, key, src_path):
         pass
 
-    def as_df(self, key, columns=None, df_module=None, format="", **kwargs):
+    def as_df(self, url, subpath, columns=None, df_module=None, format="", **kwargs):
```

Comment:
This is a backwards incompatible change, why not leaving the name as is and adding the `subpath` as the last kwarg ?

Reply:
its internal (used only once), the user facing func is DataItem.as_df(), which calls to store.as_df()